resources/public/api/conf/4.0/loss_claims_endpoints.yaml

```diff
@@ -1,17 +0,0 @@
-post:
```

Comment:
As the list endpoint was moved to a new path, only create remains here. This meant we no longer need the intermediate file.